src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala

```diff
@@ -58,18 +58,18 @@ class ConnectionActor(
 
     case Event(_: AmqpShutdownSignal, _) => stay()
 
-    case Event(msg @ ProvideChannel, _) =>
-      log.debug("{} can't create channel for {} in disconnected state", header(Disconnected, msg), sender())
+    case Event(ProvideChannel, _) =>
+      log.debug("{} can't create channel for {} in disconnected state", header(Disconnected, ProvideChannel), sender())
       stay()
   }
   when(Connected) {
-    case Event(msg @ ProvideChannel, Connected(connection)) =>
+    case Event(ProvideChannel, Connected(connection)) =>
       safe(connection.createChannel()) match {
         case Some(channel) =>
-          log.debug("{} channel acquired", header(Connected, msg))
```

Comment:
According to compiler it is. If you checkout commit right before that. Compiler will give this nice warning
/.../akka-rabbitmq/src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala:61: The value matched by ConnectionActor.ProvideChannel is bound to msg, which may be used under the unsound assumption that it has type com.thenewmotion.akka.rabbitmq.ConnectionActor.ProvideChannel.type, whereas we can only safely count on it having type Any, as the pattern is matched using `==` (see SI-1503).